lib/internal/errors.js

```diff
@@ -229,6 +229,8 @@ E('ERR_NAPI_CONS_PROTOTYPE_OBJECT', 'Constructor.prototype must be an object');
 E('ERR_NO_CRYPTO', 'Node.js is not compiled with OpenSSL crypto support');
 E('ERR_NO_ICU', '%s is not supported on Node.js compiled without ICU');
 E('ERR_NO_LONGER_SUPPORTED', '%s is no longer supported');
+E('ERR_OPT_VALUE_OUT_OF_RANGE', (name, expected, actual) =>
+  `The value of "${name}" must be ${expected}. Received "${actual}"`);
```

Comment:
To keep consistency it would be best to use `%s` and rely on `util.format` instead of using a function.